### ISSUE TITLE
Enunciate Ant task failing -  no source files or class names

### ISSUE THREAD

**Issue by sdebacq** [stoicflame/enunciate#100]
I've tried to switch to enunciate v2 using Ant, but I'm stumbling on an error:
```
java.lang.IllegalStateException: no source files or class names
	at com.sun.tools.javac.main.Main.error(Main.java:186)
	at com.sun.tools.javac.main.Main.compile(Main.java:426)
	at com.sun.tools.javac.api.JavacTaskImpl.doCall(JavacTaskImpl.java:129)
	at com.sun.tools.javac.api.JavacTaskImpl.call(JavacTaskImpl.java:138)
	at com.webcohesion.enunciate.Enunciate.run(Enunciate.java:585)
	at com.webcohesion.enunciate.EnunciateTask.execute(EnunciateTask.java:143)
	at org.apache.tools.ant.UnknownElement.execute(UnknownElement.java:293)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.lang.reflect.Method.invoke(Method.java:497)
	at org.apache.tools.ant.dispatch.DispatchUtils.execute(DispatchUtils.java:106)
	at org.apache.tools.ant.Task.perform(Task.java:348)
	at org.apache.tools.ant.Target.execute(Target.java:435)
	at org.apache.tools.ant.Target.performTasks(Target.java:456)
	at org.apache.tools.ant.Project.executeSortedTargets(Project.java:1405)
	at org.apache.tools.ant.Project.executeTarget(Project.java:1376)
	at org.apache.tools.ant.helper.DefaultExecutor.executeTargets(DefaultExecutor.java:41)
	at org.apache.tools.ant.Project.executeTargets(Project.java:1260)
	at org.apache.tools.ant.Main.runBuild(Main.java:853)
	at org.apache.tools.ant.Main.startAnt(Main.java:235)
	at org.apache.tools.ant.launch.Launcher.run(Launcher.java:285)
```
Here's a simple dummy project, using one of the enunciate examples, where I recreated the issue:
https://dl.dropboxusercontent.com/u/1040158/enunciate-example.zip

Is this a bug or did I make a mistake in the configuration.

**Comment by stoicflame** [comment]
Thanks for the example and the report. I'll take a closer look and cut an M.2 release.

**Comment by stoicflame** [comment]
This was fixed at 854d52f.
